templates/house_view/house_create.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %} {% load static %}
+{% load crispy_forms_tags %}
+{% block content %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/house_create.css' %}"/>
+    <div id="house_create">
+        <div class="d-flex justify-content-center h-100">
+            <div class="card">
+                <div class="card-header">
+                    <h3>Add your House information</h3>
+                </div>
+                <div class="card-body" style="color: rgb(0, 0, 0); background: rgba(252, 252, 252, 0.3); width: 500px;">
+                    <form method="POST">
+                        {% csrf_token %}
+                        <fieldset class="form-group">
+                            {{ form|crispy }}
+                        </fieldset>
+                        <button class="btn btn-primary" id="submit-button" type="submit">Submit</button>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock content %}
```

Comment:
You need to add the city by country select logic we have in our global form,
maybe we can export that js logic to a file so you can use it here easily and so that the city select would change when a country is selected

Reply:
I'll work on it in a different PR @shlior7 👍

Reply:
Thanks @LiorKGOW  #114